src/index.test.js

```diff
@@ -53,5 +53,6 @@ test('works with iopipe', async function runTest() {
   expect(val).toBe('wow');
   expect(profilerRuntime.running).toBe(false);
   expect(putData.length).toBe(1);
-  expect(putData[0]).toBe('woot');
+  // Test that the data returned has the zip format magic bytes.
+  expect(putData[0].slice(0, 4)).toEqual(Buffer([80, 75, 3, 4]));
```

Comment:
we could look inside the zip file and check the contents too

Reply:
We should replicate this test with the profiler enabled, the heap enabled, and with both enabled.

Maybe replicate this test with everything disabled as well! Then we know a disabled plugin doesn't break invocations.